1. Declare two variables, `x` and `y`, and assign them integer values. 
Swap the values of these variables without using any temporary variable.

In [1]:
x= 5 ; y= 9

In [2]:
x,y

(5, 9)

In [3]:
x,y = y,x

In [4]:
x,y

(9, 5)

RESULT1: Hence the values Swapped

2. Create a program that calculates the area of a rectangle. 
Take the length and width as inputs from the user and store them in variables. 
Calculate and
display the area.

In [5]:
length = float(input())

 4.1


In [7]:
width = float(input())

 2.8


In [8]:
length,width

(4.1, 2.8)

In [9]:
area = length *width

In [10]:
area

11.479999999999999

RESULT2: The area of Rectangle with lenth and Width of :  4.1, 2.8  respectively is  11.4799999999.

3. Write a Python program that converts temperatures from Celsius to Fahrenheit. 
Take the temperature in Celsius as input, store it in a variable,
convert it to Fahrenheit, and display the result.

Formula :Temperature in degrees Fahrenheit (°F) = (Temperature in degrees Celsius (°C) * 9/5) + 32

In [11]:
cel = float(input())

 100


In [12]:
cel

100.0

In [13]:
fah = cel*(9/5) +32
print(fah)

212.0


RESULT3 : 100 celsius = 212  Fahrenheit

TOPIC: String Based Questions

1. Write a Python program that takes a string as input and prints the length of the string.

In [14]:
s = str(input())

 Is Python a hard language to learn?.


In [15]:
s

'Is Python a hard language to learn?.'

In [16]:
len(s)

36

2. Create a program that takes a sentence from the user and counts the number of vowels (a, e, i, o, u) in the string.

In [17]:
inputs = str(input())

 Everybody is happy


In [18]:
vowels = set("aeiouAEIOU")
count =0
vowels          

{'A', 'E', 'I', 'O', 'U', 'a', 'e', 'i', 'o', 'u'}

In [19]:
for s in inputs:
    if s in vowels:
        count = count + 1
        print(count)

1
2
3
4
5


RESULT: Using For and If Function : The number of vowels is counted.

3. Given a string, reverse the order of characters using string slicing and print the reversed string.

In [20]:
rev = "hieveryone"

In [21]:
rev

'hieveryone'

In [22]:
rev[::-1]

'enoyreveih'

RESULT : Hence Input String has been reversed

4. Write a program that takes a string as input and checks if it is a palindrome
(reads the same forwards and backwards).

In [23]:
front = str(input())

 level


In [24]:
back = front[::-1]
back

'level'

In [25]:
if front == back:
    print("Palindrome")
else:
    print("OOPS no palindrome")

Palindrome


RESULT: level is a Palindrome

In [26]:
a = str(input())

 goat


In [27]:
b = a[::-1]
b

'taog'

In [28]:
if b ==a:
    print("Yes it is Palindrome")
else:
    print("NO, It isnt Palindrome")

NO, It isnt Palindrome


RESULT: goat isnt a Palindrome

5. Create a program that takes a string as input and removes all the spaces from it. Print the modified string without spaces.

Using a Funtion

In [32]:
def rem_space(hello):
    return hello.replace(" ","")
    

In [33]:
hello = str(input())

 h o w is your P Y T HON cla ss es go ing  ?


In [34]:
print(rem_space(hello))

howisyourPYTHONclassesgoing?


Using Replace

In [35]:
y = str(input())

 be st of lu ck !! !! :)


In [36]:
w = y.replace(" ","")
w

'bestofluck!!!!:)'